examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+       "1         0.282609            0.326087                          NaN   \n",
+       "2         0.266667            0.266667                          3.0   \n",
+       "3         0.247191            0.314607                          4.0   \n",
+       "4         0.225000            0.250000                          4.0   \n",
+       "5         0.061538            0.061538                          3.0   \n",
+       "6         0.274510            0.333333                          3.0   \n",
+       "7         0.192771            0.192771                          3.0   \n",
+       "8         0.253521            0.281690                          4.0   \n",
+       "9         0.290909            0.327273                          3.0   \n",
+       "\n",
+       "                 answer_correctness/v1/justification  answer_quality/v1/score  \\\n",
+       "0  The output provided by the model is mostly cor...                        3   \n",
+       "1  Failed to extract score and justification. Raw...                        4   \n",
+       "2  The output provided by the model addresses a c...                        3   \n",
+       "3  The output provided by the model is mostly cor...                        4   \n",
+       "4  The output provided by the model is mostly cor...                        4   \n",
+       "5  The output provided by the model addresses a c...                        3   \n",
+       "6  The output provided by the model addresses a c...                        4   \n",
+       "7  The output provided by the model addresses a c...                        4   \n",
+       "8  The output provided by the model is mostly cor...                        4   \n",
+       "9  The output provided by the model addresses a c...                        4   \n",
+       "\n",
+       "                     answer_quality/v1/justification  \n",
+       "0  The output is understandable and fluent, but i...  \n",
+       "1  The output is fluent and clear, as it reads sm...  \n",
+       "2  The output is understandable but lacks some im...  \n",
+       "3  The output is fluent and clear, as it is easy ...  \n",
+       "4  The output is fluent and clear, as it reads sm...  \n",
+       "5  The output is understandable but still needs i...  \n",
+       "6  The output is fluent and clear, as it reads sm...  \n",
+       "7  The output is fluent and clear, providing a co...  \n",
+       "8  The output is fluent and clear, as it reads sm...  \n",
+       "9  The output is fluent and clear, as it is easy ...  "
+      ]
+     },
+     "execution_count": 50,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results.tables[\"eval_results_table\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we can view our evaluation results in the MLflow UI under the Evaluation tab. Here, we can choose which columns to group by and a column to compare on."
```

Comment:
Let's show a few more screenshots to demonstrate how to jump from the landing page to the evaluation tab, I feel it's not quite trivial.